packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/components/ConsensusPopup/ConsensusPopup.js

```diff
@@ -13,7 +12,7 @@ counterpart.registerTranslations('en', en)
 const StyledBox = styled(Box)`
   font-family: "Roboto Mono Regular", monospace;
 
-  ${List} {
+  ul {
```

Comment:
I wonder if we need to update styled-components

Reply:
per the [styled-components docs](https://styled-components.com/docs/api#isstyledcomponent) it should have been fine, no related issues popping up after cursory review, but I could be missing something related. I can make an issue to update styled-components? Or not sure issue warranted (should be updating all deps).